great_expectations/expectations/core/expect_column_quantile_values_to_be_between.py

```diff
@@ -57,6 +57,11 @@
     from great_expectations.render.renderer_configuration import AddParamArgs
 
 
+class QuantileRange(TypedDict):
```

Comment:
Using the class syntax. Looks like we used the "Alternative" syntax for a different TypedDict, but AFAIK, that's only needed to support python < 3.6